app/src/main/java/com/tip/lunchbox/data/util/RetrofitSingleton.java

```diff
@@ -0,0 +1,36 @@
+package com.tip.lunchbox.data.util;
+
+import okhttp3.OkHttpClient;
+import retrofit2.Retrofit;
+import retrofit2.adapter.rxjava3.RxJava3CallAdapterFactory;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+public class RetrofitSingleton {
```

Comment:
Nice work! but here are few things that need attention:
* The class name should follow the pattern `%Util`. So in this case, ideally the class should be named `RetrofitUtil`
* Util classes should be declared _final_
* All variables and methods should ideally be _static_.

Reply:
Check now